dojo/settings/settings.dist.py

```diff
@@ -1109,7 +1109,7 @@ def saml2_attrib_map_format(dict):
 if len(env("DD_CELERY_BROKER_TRANSPORT_OPTIONS")) > 0:
     CELERY_BROKER_TRANSPORT_OPTIONS = json.loads(env("DD_CELERY_BROKER_TRANSPORT_OPTIONS"))
 
-CELERY_IMPORTS = ("dojo.tools.tool_issue_updater", )
+CELERY_IMPORTS = ("dojo.tools.tool_issue_updater", "dojo.problem.tasks")
```

Comment:
Maybe rename the file to something like `dojo.problem.update_mappings` to make what it does clearer.